src/cider/function/scalar/RuntimeFunctions.cpp

```diff
@@ -1075,6 +1075,7 @@ extern "C" ALWAYS_INLINE uint64_t string_pack(const int8_t* ptr, const int32_t l
 }
 
 #ifdef __clang__
+#include "function/string/CiderStringFunctions.cpp"
```

Comment:
Better to put these new functions under nextgen/function folder?

Reply:
updated.